R/io_csv.R

```diff
@@ -134,7 +134,9 @@ pl_scan_csv = function(
         return(RPolarsRNullValues$new_named(unlist(null_values)))
       }
 
-      stop("null_values arg must be a string OR unamed char vec OR named char vec")
+      Err_plain(
+        "null_values arg must be a string OR unamed char vec OR named char vec"
+      ) |> unwrap()
```

Comment:
Could you use `unwrap()` with the context argument?
Patterns used elsewhere in the package, such as:

```r
uw = function(res) unwrap(" in pl$scan_csv():")
```

Reply:
Sure thing. What about the internal functions i.e. `check_is_link` are you happy with `unwrap("in check_is_link():")` or should this be left alone as it is internal and not exported to the user.

Reply:
For internal functions like `check_is_link()`, you can leave `unwrap()` as is. This is what we do for other internal functions